README.md

```diff
@@ -40,22 +30,27 @@ Debian and derived distributions would be:
 sudo apt install libxml2-dev
 ```
 
-#### From the source code
+## Usage
 
-```bash
-git clone https://github.com/bblfsh/client-python.git
-cd client-python
-make install
-```
+A small example of how parse a Python file and extract the import declarations from the [UAST](https://doc.bblf.sh/uast/specification.html).
 
-#### Using pip3
+If you don't have a bblfsh server running you can execute it using the following command:
 
-```bash
-pip3 install bblfsh
+```sh
+docker run --privileged --rm -it -p 9432:9432 --name bblfsh bblfsh/server
 ```
 
-It is possible to regenerate the gRPC/protobuf bindings by executing `make`.
+Please read the [getting started](https://doc.bblf.sh/user/getting-started.html) guide, to learn more about how to use and deploy a bblfsh server.
```

Comment:
Comma  after "Please" (optional, gives more emphasis to the please), remove it after "guide".